app/pages/collections/nav.cjsx

```diff
@@ -0,0 +1,45 @@
+React = require 'react'
+{Link, IndexLink} = require 'react-router'
+Translate = require 'react-translate-component'
+classNames = require 'classnames'
+
+CollectionsNav = React.createClass
+  displayName: 'CollectionsNav'
+
+  contextTypes:
+    geordi: React.PropTypes.object
+
+  componentWillReceiveProps: (nextProps, nextContext)->
+    @logClick = nextContext?.geordi?.makeHandler? 'collect-menu'
+
+  render: ->
+    classes = classNames {
+      "about-tabs": @props.project?
+    }
+    baseLink = ""
+    messageKeyPrefixToUse=""
```

Comment:
Could you add spaces around `=` in the assignment here? My text editor thinks everything after it is a string.